ietf/home/templates/includes/post_iab.html

```diff
@@ -0,0 +1,32 @@
+{% load wagtailimages_tags %}
+
+<li class="card mb-4 row g-0 align-items-stretch iab-blog-post">
+    <div>
+        {% if post.feed_image and show_image %}
+            {% image post.feed_image height-245 as post_image %}
+            <img class="blog-post-image-iab u-obj-cover w-100 h-100" src="{{ post_image.url }}" alt="{{ post.main_image.title }}">
+        {% endif %}
+    </div>
+    {# col-* classes include positioning, position static cancels that and allows the stretched link to cover the image #}
+    <div class="card-body card-iab">
+        <h3 class="mb-3 h5"><a class="stretched-link" href="{{ post.url }}">{{ post.title }}</a></h3>
+        <p class="card-text-iab">{{ post.introduction }}</p>
+        <div class="d-flex justify-content-between align-items-center date-text-iab">
+            <small>
```

Comment:
I haven't seen the `<small>` tag used in years, though I can't see anywhere saying not to use it. It's semantically fine. Though if you wanted more fine-grained control over what size this actually renders at (to match the design) I'd suggest just adding BEM classes to the `<p>`s.

Reply:
cool, leaving it as is to stay more consistent with IETF